### DISCUSSION TITLE
Textarea losing focus

### DISCUSSION THREAD

**Discussion by kaykegaldiano** [livewire/livewire#6456] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.10

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/kaykegaldiano/livewire-test

### Steps To Reproduce

I have created a full page component named Home

```
class Home extends Component
{
    #[Rule('required|min:3')]
    public string $username;

    #[Rule('required|min:10')]
    public string $description;

    #[Title('Home')]
    public function render()
    {
        return view('livewire.home');
    }

    public function displayData()
    {
        $this->validate();

        dd($this->username);
    }
}
```

Here's the view:
```
<div class="flex flex-col items-center pt-6 space-y-4">
    <h1 class="font-bold text-2xl">Testing input and textarea</h1>

    <div>
        <x-input-label value="Username"/>
        <x-input type="text" name="username" wire:model.live='username' />
        {{ $username }}
        <x-input-error :messages="$errors->get('username')" />
    </div>

    <div>
        <x-input-label value="Description"/>
        <textarea
            name="description"
            class="block w-full rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 sm:text-sm sm:leading-6"
            wire:model.live='description'></textarea>
            {{ $description }}
        <x-input-error :messages="$errors->get('description')" />
    </div>

    <div>
        <button
            class="flex w-full justify-center rounded-md bg-indigo-600 px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
            wire:click='displayData'>Send</button>
    </div>
</div>
```

The problem I'm experiencing is that when I type if I put wire:model.live in a textarea and start typing, it loses focus, but when I do the same in an input it works fine without losing focus.

No errors showing on console or PHP. What could it be? It is supposed to work like that? I'm new to Livewire, so I don't know it that is normal or not.

You can reproduce my steps cloning my repository.

Thanks!

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by zbora23** [comment]
I am experiencing the exact same issue, seems like `<input type="text">` behaves differently to `<textarea>` element

**Reply by kaykegaldiano** [reply]
Yes, I've just tested it and the problem persists with the textarea with live update. It's weird because I tested this with Livewire 2 and it works normal

**Comment by kaykegaldiano** [comment]
Well, I've just found a way to make the textarea work with `wire:model.live` without lose focus:

```html
<div wire:ignore>
  <textarea wire:model.live="info" id="info" rows="4"></textarea>
</div>
``` 
Put the textarea inside a div and then add wire:ignore in that div, this way the textarea won't lose focus.
